### PR TITLE
refact: page layouts in the Oracle section

### PR DESCRIPTION
**Pull request by martinvibes**
Ready for review

- - -
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/970)

### CODE REVIEW

File: src/ch16-05-01-price-feeds.md [hunk 1]
```diff
@@ -15,19 +17,19 @@ To get started with integrating Pragma on your Cairo smart contract for price fe
 pragma_lib = { git = "https://github.com/astraly-labs/pragma-lib" }
 ```
 
-## Creating a Price Feed Contract
+### Creating a Price Feed Contract
 
 After adding the required dependencies for your project, you'll need to define a contract interface that includes the required pragma price feed entry point.
 
-### Contract Interface
+### Define the Contract Interface
```
Comment (enitrat) [inline src/ch16-05-01-price-feeds.md:24]: I think both "Define the Contract Interface" and "Import Pragma Dependencies" are superfluous as subsections, we don't need special headers as they can just be part of the parent `### Creating a Price Feed Contract`

```suggestion
```

File: src/ch16-05-02-randomness.md [hunk 2]
```diff
@@ -4,9 +4,11 @@ Since all blockchains are fundamentally deterministic and most are public ledger
 
 ## Overview on VRFs
 
-Pseudo-random but secure: VRFs use a secret key and a nonce (a unique input) to generate an output that appears random. While technically 'pseudo-random', it's practically impossible for another party to predict the outcome without knowing the secret key.
+### Pseudo-random but secure
```
Comment (enitrat) [inline src/ch16-05-02-randomness.md:7]: These two subheaders can be removed, leaving one single paragraph for the overview of VRFs

```suggestion
```

**Comment by martinvibes** [comment]
@enitrat  suggested changes have been implemented.

**Review by enitrat — APPROVED** [review]

**Comment by martinvibes** [comment]
Hello @enitrat thanks for the merge

**Pull request merged**